docs/troubleshooting/windows-pv-tools.md

```diff
@@ -60,4 +60,19 @@ It's possible that some antivirus blocks the end of the installation of the PV d
 
 ### Solution
 
-Simply pausing the agent and doing a reboot will make the XenTools to install succesfully. After a succesfull installation, enabling the SentinelOne agent again is possible without any other issues regarding the tools or drivers.
\ No newline at end of file
+Simply pausing the agent and doing a reboot will make the XenTools to install succesfully. After a succesfull installation, enabling the SentinelOne agent again is possible without any other issues regarding the tools or drivers.
+
+## Windows bug check 0x3B (SYSTEM_SERVICE_EXCEPTION) on systems with newer Intel CPUs
+
+### Cause
+
+Intel CPUs codenamed Rocket Lake, Sapphire Rapids and newer provide the **Architectural LBR** feature, which Windows depends on.
+Xen's support of this CPU feature is incomplete, which causes Windows to crash.
+
+### Workaround
+
+Run the following command on the host then stop and restart the VM:
+
+```
+xe vm-param-add uuid=<VM's UUID> param-name=platform msr-relaxed=true
+```
```

Comment:
Are there security implications we ought to mention?

Reply:
I couldn't think of one from the viewpoint of XCP-ng. It leaks only the list of readable MSRs on the host (not their contents). Xen maintainers may know some further implications.